apps/src/blockly/addons/contextMenu.js

```diff
@@ -184,6 +185,36 @@ const registerUnshadow = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(unshadowOption);
 };
 
+const registerKeyboardNavigation = function() {
+  const keyboardNavigationOption = {
+    displayText: function(scope) {
+      const displayText = scope.workspace.keyboardAccessibilityMode
+        ? msg.blocklyKBNavOff()
+        : msg.blocklyKBNavOn();
+      return displayText;
+    },
+    preconditionFn: function() {
+      if (Blockly.navigationController) {
+        return 'enabled';
+      } else {
+        console.log('navigationController not found');
```

Comment:
should this log be here?

Reply:
Thanks! After removing this log, I was able to compress 5 lines down to 1 by using a ternary.:)